docs/concept-exercises.md

```diff
@@ -58,14 +58,14 @@ An exercise has the following files. In the browser they will show at the releva
 **Purpose:** Introduce the concept(s) that the exercise teaches to the student.
 
 - The information provided should give the student just enough context to figure out the solution themselves.
-- Links should be used sparingly if at all. While a link explaining a complex topic like recursion might be useful, for most concepts the links will provide more information than needed so explaining things concisely inline should be the aim.
-- The file should not give the student any extra information about a concept not needed to understand the fundamentals of the concept or solve the exercise.
-- Use the proper technical terms so that the student can easily search for more information.
-- If the exercise introduces new syntax, an example of the syntax should always be included; students should not need to search the web for examples of syntax.
+- Don't give any extra information about a concept not needed to understand the fundamentals of the concept or solve the exercise.
+- Links should be used sparingly, if at all. While a link explaining a complex topic like recursion might be useful, for most concepts the links will provide more information than needed so explaining things concisely inline should be the aim.
+- Use the proper technical terms (e.g. a keyword) so that the student can easily search for more information.
+- Don't use code examples unless the exercise introduces new syntax, in which case an example of the syntax must be included; students should not need to search the web for examples of syntax.
 
 As an example, the introduction to a "strings" exercise might describe a string as just a "Sequence of Unicode characters" or a "series of bytes", tell the users how to create a string, and explain that a string has methods that can be used to manipulate it. Unless the student needs to understand more nuanced details in order to solve the exercise, this type of brief explanation (along with an example of its syntax) should be sufficient information for the student to solve the exercise.
 
-See the C# floating-point-numbers exercise's [introduction.md file][csharp-docs-introduction.md] for an example.
+See [this example introduction.md file][introduction.md]. Notice how the introduction is very minimal, but that the language-specific keywords are enquoted and an example of the newly introduced syntax is included.
```

Comment:
"enquoted" is not a very common term IME. Even google suggests "Did you mean _inputted_?". Non-native speakers may struggle with it.

```suggestion
See [this example introduction.md file][introduction.md]. Notice how the introduction is very minimal, but that the language-specific keywords are enclosed in quotes and an example of the newly introduced syntax is included.
```